mathml/presentation-markup/scripts/empty-underover.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test empty munder, mover and munderover</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#underscripts-and-overscripts-munder-mover-munderover">
+<meta name="assert" content="Elements munder, mover, munderover should not render anything when empty.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/mathml/support/feature-detection.js"></script>
+<script>
+  var epsilon = 1;
+
+  function getBox(aId) {
+    return document.getElementById(aId).getBoundingClientRect();
+  }
+
+  setup({ explicit_done: true });
+  window.addEventListener("load", () => { document.fonts.ready.then(runTests); });
+
+  function runTests() {
+    test(function() {
+      assert_true(MathMLFeatureDetection.has_mspace());
+      assert_approx_equals(getBox("underover").width, 0, epsilon, "width of empty munderover");
+      assert_approx_equals(getBox("underover").height, 0, epsilon, "height of empty munderover");
+    }, "Empty munderover");
+
+    test(function() {
+      assert_true(MathMLFeatureDetection.has_mspace());
+
+      assert_approx_equals(getBox("under").width, 0, epsilon, "width of empty munder");
+      assert_approx_equals(getBox("under").height, 0, epsilon, "height of empty munder");
+    }, "Empty munder");
+
+    test(function() {
+      assert_true(MathMLFeatureDetection.has_mspace());
+
+      assert_approx_equals(getBox("over").width, 0, epsilon, "width of empty mover");
+      assert_approx_equals(getBox("over").height, 0, epsilon, "height of empty mover");
+    }, "Empty mover");
```

Comment:
It seems you can generate these tests with a loop

["over", "under", "underover"].forEach

Probably do the same for sub/sup/subsup/mmultiscripts.

And we should more generally do tests for how to handle invalid markup #15